en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,37 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+In some case, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page is not enough. There are some new behaviour to add to the form, such as a specific event
+when you edit a field.
```

Comment:
After this, I would quickly explain that it's possible to fully manage the view from the listing page, but if the dev wants to use the orchestra listing view and yet change the edit action, then here's the way to go.